### DISCUSSION TITLE
Compile Error When Trying to use Arduino IDE (not portable)

### DISCUSSION THREAD

**Discussion by redchairlabs** [sanni/cartreader#1005] [General Chat]
I'm still a bit new to Arduino but I I like to do my editing in my installed IDE (comfort zone). So I copied the sketchbook found in the v14 Portable download [(directions here)](https://github.com/sanni/cartreader/wiki/How-to-flash-the-Arduino#official-arduino-ide-installer).

When compiling I get errors indicating that it can't find various files or libraries like this:

*****************************************************
`In file included from C:\Users\jwillette\Documents\Arduino\sketchbook\Cart_Reader\Cart_Reader.ino:65:0:
C:\Users\jwillette\Documents\Arduino\sketchbook\Cart_Reader\OSCR.h:13:10: fatal error: SdFat.h: No such file or directory
 #include "SdFat.h"
          ^~~~~~~~~
compilation terminated.
exit status 1

Compilation error: SdFat.h: No such file or directory`

*****************************************************

I get that it can't find the "SdFat.h" and other files that are being included in the sketch but I'm not sure how to tell the IDE where they are.

When I compile and load using teh protable version, everything works just fine. However I woudl really like tot get this to work from My IDE instead.

Any help would be appreciated.

**Comment by sanni** [comment]
Copy like this:
C:\Users\jwillette\Documents\Arduino\Cart_Reader
C:\Users\jwillette\Documents\Arduino\libraries

Not like this:
C:\Users\jwillette\Documents\Arduino\sketchbook\Cart_Reader
C:\Users\jwillette\Documents\Arduino\sketchbook\libraries

**Reply by redchairlabs** [reply]
I'm such an idiot. Thank You!
